### ISSUE TITLE
Colored shape Reducing Circle Accuracy Slider to 0 causes exceptions every loop

### ISSUE THREAD

**Issue by gerth2** [PhotonVision/photonvision#403]
Stack trace was not put to log either:

```
[2022-01-16 15:57:10] [VisionModule - VisionRunner - USBFrameProvider - opencv_Microsoft_LifeCam_HD-3000] [ERROR] Exception on loop 84804
[2022-01-16 15:57:10] [VisionModule - VisionRunner - USBFrameProvider - opencv_Microsoft_LifeCam_HD-3000] [ERROR] Exception on loop 84805
[2022-01-16 15:57:10] [VisionModule - VisionRunner - USBFrameProvider - opencv_Microsoft_LifeCam_HD-3000] [ERROR] Exception on loop 84806
[2022-01-16 15:57:10] [VisionModule - VisionRunner - USBFrameProvider - opencv_Microsoft_LifeCam_HD-3000] [ERROR] Exception on loop 84807
```

**Comment by mcm001** [comment]
Fix is to make the min 1 in the UI

**Comment by mcm001** [comment]
We may wanna make a backend fix to resolve this. but this shouldn't happen with the current frontend only fix

**Comment by mcm001** [comment]
Closed by #688
